nuget/Microsoft.Windows.CsWinRT.nuspec

```diff
@@ -26,5 +26,7 @@
     <file src="readme.txt"/>
     <file src="$net5_runtime$" target="lib\net5.0\"/>
     <file src="$source_generator$" target="analyzers\dotnet\cs\"/>
+    <file src="$winrt_host$" target ="lib"/>
```

Comment:
should the target be lib\net5.0\ like the runtime dll ?

Reply:
winrt.host is a native dll so possibly under native.